keyboards/brick/brick.c

```diff
@@ -1,5 +1,4 @@
-/*
- * Copyright 2020 Jason Chestnut <jason.chestnut@gmail.com>
+/* Copyright 2023 Juno Nguyen <juno.ngx@gmail.com>
```

Comment:
This is not how copywrite and ownership change should be handled on original files with other people's names on it